UnityProject/Assets/Scripts/Objects/Closets/AbandonedCrates.cs

```diff
@@ -0,0 +1,67 @@
+using System.Collections.Generic;
+using MiniGames;
+using Mirror;
+using UnityEngine;
+
+namespace Objects.Closets
+{
+	public class AbandonedCrates : ClosetControl, IMiniGame
+	{
+
+		[SerializeField] private MiniGameResultTracker miniGameTracker;
+		[SerializeField] private List<MiniGameModule> miniGameModules;
+		[SyncVar] private int currentMiniGameIndex = -1;
```

Comment:
Why is this synced to the client?

Reply:
bi-product of old code while testing, removing.